hazelcast-enterprise/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM alpine:3.11
+FROM registry.redhat.io/ubi8/ubi
 
 # Versions of Hazelcast and Hazelcast plugins
 ARG HZ_VERSION=4.0.3
 ARG CACHE_API_VERSION=1.1.1
 ARG JMX_PROMETHEUS_AGENT_VERSION=0.11.0
-ARG NETTY_VERSION=4.1.44.Final
-ARG NETTY_TCNATIVE_VERSION=2.0.28.Final
+ARG NETTY_VERSION=4.1.47.Final
```

Comment:
Have you tested if OpenSSL works correctly with the updated versions?

Reply:
If we are updating the lib, we should switch to even newer version `4.1.53.Final` and the latest tcnative: `2.0.34.Final`. They are currently used in the hazelcast-enterprise `master` branch.

Reply:
All Netty dependencies were already removed at master branch so all of them will be gone when the PR rebased.